apps/librepcb/main.cpp

```diff
@@ -82,6 +82,24 @@ int main(int argc, char* argv[])
     // Initialize all 3rd party libraries
     init3rdPartyLibs();
 
+    // Migrate from old setting format to INI
+    // backward compatibility - remove this some time!
+    QSettings::setDefaultFormat(QSettings::NativeFormat);
+    QSettings oldSettings;
+    QSettings::setDefaultFormat(QSettings::IniFormat);
+    QSettings newSettings;
+    if (!FilePath(newSettings.fileName()).isExistingFile()) {
+        qInfo() << "Migrating old settings to" << newSettings.fileName();
+        foreach (const QString& key, oldSettings.allKeys()) {
+            newSettings.setValue(key, oldSettings.value(key));
+        }
+    }
+
+    // Make sure the INI format is used for settings on all platforms because:
+    // - Consistent storage format on all platforms
+    // - Windows Registry is bullshit
```

Comment:
Maybe use less profane language :laughing: Something like: .ini files are easily version controlled compared to registry settings.

Reply:
+1 for bullshit :-D

Reply:
I could add a ";)" to make clear that it is meant to amuse the reader :)